types/chrome/index.d.ts

```diff
@@ -10019,14 +10028,14 @@ declare namespace chrome.vpnProvider {
 declare namespace chrome.wallpaper {
     export interface WallpaperDetails {
         /** Optional. The jpeg or png encoded wallpaper image. */
-        data?: any;
+        data?: ArrayBuffer;
```

Comment:
https://developer.chrome.com/docs/extensions/reference/wallpaper/#method-setWallpaper

Reply:
Since the `exactOptionalPropertyTypes` typescript flag is set, undefined should be explicitly declared.
```suggestion
        data?: ArrayBuffer | undefined;
```